pkg/lockfile/parse.go

```diff
@@ -32,6 +32,7 @@ var parsers = map[string]PackageDetailsParser{
 	"yarn.lock":                   ParseYarnLock,
 	"gradle.lockfile":             ParseGradleLock,
 	"buildscript-gradle.lockfile": ParseGradleLock,
+	"installed": 				   ParseApkInstalled,
```

Comment:
It's probably not worth the extra complexity to try and solve right now, but I'm interested in how others feel about this - while I know `installed` is the name of the file, it feels a little too generic; I wouldn't be surprised if there's another tool out there that creates a file with the same name that we might one day want to support.

Reply:
I agree it's too generic, it is not unlikely someone might just have a `installed` file in their repository completely unrelated to alpine, which will spit out potentially confusing warnings. 

A quick fix could be to check whether `installed` is in `/lib/apk/db/installed` and only scan that, though that is quite limited, since you might be copying it out, or mounting a docker image ...etc.

Reply:
I wonder if this should be handled similar to sboms and csvs in `osv-detector` (which are a parser located in `lockfile`, but are never assumed when scanning directories - you have to explicitly say "this is a CSV I want you to check") - because `/lib/apk/db/installed` is very specific, so you're only ever going to get it checked if you run the scanner from anywhere along that path.

So what if instead it had a dedicated flag, like `--check-apk-installed`?

Reply:
That's a good idea, I made an issue here #124 to track it since as you said it's beyond the scope of this PR. We probably should solve this before the next release to avoid changing behavior.